README.md

```diff
@@ -2,6 +2,16 @@
 
 An website for watching videos together.
 
+## Quick Start
+
+1. Clone this repo via `git clone git@github.com:howardchung/watchparty.git`
+2. Install npm dependencies for the project via `npm install`
+3. Start the server via `npm run dev`
```

Comment:
might want to add some notes on the PORT env var. Currently either the REACT_APP_SERVER_HOST env var needs to be set or client defaults to port 8080 on the same host.